File: Assignment 04/Meena.py

import itertools

marbles = int(input())
queries = int(input())

arr = []
qrr = []

for marble in range(marbles):
    arr.append(input())

count = 0
crime = 0
crmea = 0

for query in range(queries):
    qrr.append(input())


for x in arr:
    for y in qrr:
        if x == y:
            count = count +1
            crime = arr.index(x)
            crmea = y

if count>0:
    print(crmea,  "found at " , crime+2)
if count ==0:
    print(crmea , "not found " )


            

File: Assignment 04/Meena.cpp
#include<cstdio>
#include<iostream>
#include<algorithm>
using namespace std;
int main()
{
    int num, que, count=0, printErJonno = 1;


    while(scanf("%d%d",&num,&que)){

        if(num==0 && que==0){
            return 0;
        }
        else{
            int marbles[num],query;

            for(int i=0; i<num; i++){
                cin >> marbles [i];
                
            }
            sort(marbles,marbles+num);

            for(int i=0; i<que; i++){
                cin >> query;

                if(i==0){
                    cout << "CASE# " << printErJonno << ":"<< endl;
                }
                count=0;

                for(int j=0; j<num; j++){

                    if(marbles[j]==query){
                        cout << query << " found at " << j+1 << endl;
                        break;
                    }
                    else{
                        count++;
                    }
                }
                if(count==num){
                    
                    cout << query << " not found" << endl;
                }
            }
        }
        printErJonno = printErJonno + 1;
    }
    return 0;
}